app/controllers/application_controller.rb

```diff
@@ -24,6 +24,14 @@ def admin_signed_in?
     session.key?(:user_id)
   end
 
+  def admin_session
```

Comment:
As with `service_operator_signed_in?`, perhaps it would be better to put this in the base admin controller.